docs/beheerhandleiding/11_cms.rst

```diff
@@ -380,8 +380,47 @@ Wanneer u op markeren klikt wordt de betreffende plugin op de pagina weergegeven
 **Creëer alias**
 Door een alias te creëren kunt u content hergebruiken op verschillende pagina’s en deze op één plek beheren. Wanneer hergebruikte content op één plek wordt gewijzigd, wordt dit door de alias ook op de andere plekken gewijzigd.
 
-11.2.9. Footer aanpassen via CMS
---------------------------------
+
+11.2.9. 'Ga Naar' links op homepagina toevoegen met CMS
+-----------------------------------------------------
+
+Wanneer u de homepagina bewerkt kunt u via het plugin menu een nieuw blok toevoegen met 1 of meerdere externe links. Elke 'Ga naar' link dient een eigen icoontje te krijgen. De iconen set die actief is in open Inwoner betreft 'Material Icons'; deze bevat meer dan 2000 icoonen die gekozen kunnen worden.
+
+Voeg eers tde plgu-in toe aa nde pagina. Voeg daarna een link toe.
```

Comment:
```suggestion
Voeg eerst de plugin toe aan de pagina. Voeg daarna een link toe.
```

Reply:
Also: mention that they add the link via clicking on the little "+" next to the link plugin. It's not obvious to everyone (you might expect to add links from within the link plugin config, but there you can only edit the name of the plugin of course).

Reply:
This PR is not up for review yet, but I'll add that too :-)